docs/source/getting_started/index.rst

```diff
@@ -1,99 +1,123 @@
 
-Let's prepare the environment
------------------------------
+Prerequisites
+-------------
+For this guide, you need a computer running  Unix-like system with ``docker`` installed. You can read how to install it on a `Docker's website <https://www.docker.com/community-edition/>`_
 
-.. hint:: Having troubles? Check FAQ section or communicate to us directly, in case you were stuck on something. We don't expect this to happen, but some issues with an environment are possible.
+.. note:: Please note that you can use Iroha without ``docker`` as well. You can read about it in other parts of documentation.
 
-Prerequisite 1
-^^^^^^^^^^^^^^
+Starting Iroha Network
+----------------------
 
-Prerequisite 2
-^^^^^^^^^^^^^^
+Creating a Docker network
+^^^^^^^^^^^^^^^^^^^^^^^^^
+To operate, Iroha requires a ``PostgreSQL`` database. Let's start with creating Docker network, so containers for Postgres and Iroha can run in the same virtual network and successfully communicate. In this guide we will call it ``iroha-network``, but you can use any name. In your terminal write following command:
 
-Prerequisite 3
-^^^^^^^^^^^^^^
+.. code-block:: shell
 
-Getting Iroha build
--------------------
+  docker network create iroha-network
 
+Starting PostgreSQL container
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Step 1
-^^^^^^
+Now we need to run ``PostgreSQL`` in a container, attach it to the network you have created before, and exposing ports for communication:
 
-Step 2
-^^^^^^
+.. code-block:: shell
 
-Step 3
-^^^^^^
+  docker run --name some-postgres \
+  -e POSTGRES_USER=postgres \
+  -e POSTGRES_PASSWORD=mysecretpassword \
+  -p 5432:5432 \
+  --network=iroha-network \
+  -d postgres:9.5
 
-Playing around with settings
-----------------------------
+Creating blockstore
+^^^^^^^^^^^^^^^^^^^
+Before we run Iroha container, we should create persistent volume to store files, storing blocks for the chain. It is done via the following command:
 
+.. code-block:: shell
 
-Step 1
-^^^^^^
+  docker volume create blockstore
 
-Step 2
-^^^^^^
+Configuring Iroha network
+^^^^^^^^^^^^^^^^^^^^^^^^^
+Now we need to configure our Iroha network. This includes creating a configuration file, generating keypairs for a users, writing list of peers and creating a genesis block. However, we have prepared an example configuration for this guide, so you can start playing with Iroha faster. In order to get those files, you need to clone the `Iroha repository <github.com/hyperledger/iroha>`_ from Github.
 
-Step 3
-^^^^^^
+.. code-block:: shell
 
-Running a single instance
--------------------------
+  git clone -b develop https://github.com/hyperledger/iroha
 
+Starting Iroha container
+^^^^^^^^^^^^^^^^^^^^^^^^
+We are ready to launch our Iroha container. Let's do it with the following command
 
-Step 1
-^^^^^^
+.. code-block:: shell
 
-Step 2
-^^^^^^
+  docker run -it --name iroha \
```

Comment:
please explain each parameter please, especially why we are using custom entrypoint :)